src/environment.ts

```diff
@@ -320,7 +324,7 @@ export class Environment {
 }
 
 function isGlobal(name: string) {
-  // @ts-ignore TS doesn't know about globalThis
+  if (name == "name") return false;
```

Comment:
I am specifically filtering out `"name"` here; it is a global (equal to the empty string), but also a very common local variable name, so we should allow it in templates. The original use of `isGlobal()` is [here](https://github.com/ventojs/vento/blob/v1.14.0/src/environment.ts#L297-L301), and changing the behavior for `"name"` fortunately doesn't break current behavior (using `|> name` would throw an error in current Vento anyway).